pkg/kubelet/network/hairpin/hairpin.go

```diff
@@ -95,6 +97,15 @@ func setUpAllInterfaces() error {
 
 func setUpInterface(ifName string) error {
 	glog.V(3).Infof("Enabling hairpin on interface %s", ifName)
-	hairpinModeFile := path.Join(sysfsNetPath, ifName, hairpinModeRelativePath)
+	ifNamePath := path.Join(sysfsNetPath, ifName)
```

Comment:
It's not really the path to `ifName`, but the path to something that represents the interface, so it should be `ifPath`